recipes-ni/ni-rtfeatures/files/init.d/handle_cpld_ip_reset

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+IP_RESET_FILE="/dev/nirtfeatures/ip_reset"
+
+IP_RESET_REQUESTED=""
+if [ -e "$IP_RESET_FILE" ]; then
+	grep -q '1' $IP_RESET_FILE && IP_RESET_REQUESTED="y"
+else
+	[ "${VERBOSE}" != "no" ] && echo "Unable to find ip_reset at normal spot."
```

Comment:
MINOR: I would prefer
```bash
[ "${verbose}" = yes ]
```
Since the variable is worded in the affirmative (as opposed to `NOT_QUIET`), we should interpret an unset `VERBOSE` as being `FALSE`.

Reply:
Done.